icees_api/utils.py

```diff
@@ -40,3 +40,10 @@ def to_qualifiers2(feature):
         }
     else:
         return feature
+
+
+def associations_have_feature_matrices(associations):
```

Comment:
This isn't going to check ALL associations for feature matrices. It'll return true after the first one it finds. Is this intended?

Reply:
@maximusunc Yes, it is intended. The reason is that pairwise associations are computed for all feature variable columns, and we only return empty query response when all associations are empty. So a valid response will return as long as there is one non-empty association feature matrix included.